app/javascript/hooks/use-theme-observer.tsx

```diff
@@ -63,13 +99,34 @@ export function useThemeObserver(): ThemeData {
         }))
       }
     }
-    mediaQueryList.addEventListener('change', mediaQueryListener)
+    const mediaQueryListAccessibilityDark = window.matchMedia(
```

Comment:
```suggestion
    const mediaQueryPrefersContrastMore = window.matchMedia(
```
...AccessibilityDark is a fantasy name, won't be clear enough on the long run